### DISCUSSION TITLE
need help again

### DISCUSSION THREAD

**Discussion by nucleusgenius9** [postalserver/postal#2233] [Help with using Postal]
I got a problem again. Emails are not coming to google mail
![wTWcLHKrqHY](https://user-images.githubusercontent.com/118116015/201757072-7d1c557f-28cc-4392-8629-90177ad659ab.jpg)

Attaching a screenshot of the error when trying to submit.

**Comment by nucleusgenius9** [comment]
it seems the problem is that SMTP is not configured.

I found one instruction https://medium.com/@smtpie/how-to-build-free-smtp-server-setup-with-postal-step-by-step-guide-dee6574367e

It says to use the service https://www.smtper.net/ is that true?

**Reply by willpower232** [reply]
It is more likely that your server provider is blocking outbound SMTP connections, you should check with them first that they allow you to send emails.

smtper.net can help you verify that your Postal server can receive incoming SMTP connections which are different.

**Comment by nucleusgenius9** [comment]
There are no restrictions on the server side, I clarified

**Reply by willpower232** [reply]
a quick google suggests this error message comes from them when either you're hitting some kind of spam rate limit or your IPv6 isn't set up correctly

https://stackoverflow.com/questions/63375834/gmail-421-4-7-0-try-again-later-closing-connection
https://serverfault.com/questions/929559/postfix-error-421-4-7-0-try-again-later-closing-connection-ehlo

**Reply by nucleusgenius9** [reply]
I tried to remove ipv6 requests, left only ipv4 available, it did not work.

**Comment by nucleusgenius9** [comment]
I'll attach some screenshots just in case.
1
![1](https://user-images.githubusercontent.com/118116015/201977911-40746c8d-97bd-48d5-bfb9-5a57b447e7bb.jpg)
2
![2](https://user-images.githubusercontent.com/118116015/201977919-dcd7b0d2-814c-44c0-9e0f-8a43e3125cb1.jpg)
3
![3](https://user-images.githubusercontent.com/118116015/201981107-852554a7-5496-46c5-aaa4-fd34b88505e2.jpg)
4
![4](https://user-images.githubusercontent.com/118116015/201981125-2df60ca6-9d3e-4be3-ab46-5d05f7a3ccc0.jpg)


The 26th port throws on the 25th port where it is necessary.
The number of letters that I send is 1 (test letter) and it does not come. So there is no spam filter involved
